### PR TITLE
`0.1.0-dev3` -> `0.1.0.dev3` to avoid confusion

### PR DESCRIPTION
**Pull request by OmarIthawi**

### CODE REVIEW

**Comment by ahmedaljazzar** [comment]
Cool idea @OmarIthawi, but this won't pass the semantic versioning rules.

**Comment by OmarIthawi** [comment]
> Cool idea @OmarIthawi, but this won't pass the semantic versioning rules.

@ahmedaljazzar it seems that I don't understand semver well.

Do you know why `pypi` thinks that we have `0.1.0.dev3`?

File: gestore/__init__.py [hunk 1]
```diff
@@ -4,4 +4,4 @@
 
 # Increase this version by 1 after every backward-incompatible
 # change in the exported data format
-__version__ = '0.1.0-dev3'
+__version__ = '0.1.0.dev3'
```
Comment (OmarIthawi) [inline gestore/__init__.py:7]: Trying again with semver.

```suggestion
__version__ = '0.1.dev3'
```

**Comment by OmarIthawi** [comment]
nevermind, semver seems to hate me today. I will not read on it and just move on to other things 😅

**Pull request closed without merging**